.run/hello world.run.xml

```diff
@@ -1,6 +1,6 @@
 <component name="ProjectRunConfigurationManager">
   <configuration default="false" name="hello world" type="Application" factoryName="Application">
-    <option name="MAIN_CLASS_NAME" value="HelloWorldApp" />
+    <option name="MAIN_CLASS_NAME" value="main.HelloWorldApp" />
```

Comment:
Yes because otherwise we cannot use it from `replay-tests/helloworld/test/ui/hello/TemplateErrorHandlerBaseSpec.java`; you cannot use a class that's not in a package, it's a Java thing, called "the default package". Not sure what purpose it serves. Here's the answer I read to decide on this solution:

https://stackoverflow.com/questions/2030148/whats-the-syntax-to-import-a-class-in-a-default-package-in-java